src/pages/BlockPage.vue

```diff
@@ -0,0 +1,169 @@
+<script setup lang="ts">
+
+import { ref, watch, onMounted, computed, toRaw } from 'vue';
+import { useRoute, useRouter } from 'vue-router';
+import { useI18n } from 'vue-i18n';
+import { indexerApi } from 'src/boot/telosApi';
+import TransactionTable from 'components/TransactionTable.vue';
+import BlockOverview from 'components/BlockOverview.vue';
+import { BlockData } from 'src/types';
+
+
+const router = useRouter();
+const route = useRoute();
+const { t: $t } = useI18n();
+
+const defaultTab = 'overview';
+const tabs = ['overview', 'transactions'];
+
+const tab = ref(defaultTab);
+const block = ref('');
+const blockHeight = computed(() => parseInt(block.value ?? '0'));
+const blockData = ref<BlockData | null>(null);
+
+const transactionsCount = computed(() => blockData.value ? blockData.value.transactionsCount : 0);
+
+// Methods
+
+function prevBlock() {
+    router.push({ name: 'block', params: { block: parseInt(block.value) - 1 } });
+}
+
+function nextBlock() {
+    router.push({ name: 'block', params: { block: parseInt(block.value) + 1 } });
+}
+
+function visitNativeBlockExplorer(extraData) {
+    console.log('visitNativeBlockExplorer()', extraData);
+    window.open(`https://explorer-test.telos.net/block/${extraData}`, '_blank');
```

Comment:
I forgot to put this URL in the env file. I will tomorrow morning

Reply:
I took care of it